plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/Main.java

```diff
@@ -40,7 +40,7 @@ public static void main(String[] args) throws IOException {
             }
         }
 
-        System.out.println("show warnings" + options.hasShowAllWarnings());
+        System.out.println("Show warnings: " + options.isShowAllWarnings());
```

Comment:
You mean like a summary of all the CLI options they've chosen?

Reply:
I can add that; I think it might also be useful for demo purposes.  I'm going to add it using System.out.println for now, but the output would also need to be converted to Java util logging in PR 10. I think the problem of filtering out excess logging info would apply in this case as well.

Reply:
Added in ba2291f.